website/docs/language/state/examples/encryption/external-method/method-external.tofu

```diff
@@ -0,0 +1,10 @@
+terraform {
+  encryption {
+    key_provider "external" "foo" {
```

Comment:
This `key_provider` appears to be a typo, should be `method`.

Reply:
Great catch! Thanks for reporting this.
Will be fixed by #2881.